pkg/builder/builder_test.go

```diff
@@ -192,7 +210,16 @@ var _ = Describe("builder", func() {
 		}
 		c := buildPodSpec(vdb, &vdb.Spec.Subclusters[0])
 		Expect(isPasswdIncludedInPodInfo(vdb, &c)).Should(BeFalse())
+
 		vdb.Spec.StartupProbeOverride = nil
+
+		// case 2: if in vclusterops mode and version >= v23.4
```

Comment:
We can update this since we aren't going to check the version now.
```suggestion
		// case 2: if in vclusterops mode
```